modules/classicon.lua

```diff
@@ -205,12 +177,12 @@ function ClassIcon:ScanAuras(unit)
 
 	local showShortest = self.db[unit].classIconShowLowestRemainingAura
 	
-	local UnitDebuff = GladiusEx:IsTesting(unit) and ClassIcon.UnitDebuffTest or (C_UnitAuras and C_UnitAuras.GetDebuffDataByIndex or UnitDebuff)
+	local UnitDebuff = GladiusEx:IsTesting(unit) and ClassIcon.UnitDebuffTest or (C_UnitAuras and GladiusEx:UnitDebuff or UnitDebuff)

```

Comment:
. instead of : like the others, and it's also invalid syntax

Reply:
should be fixed now